playbooks/roles/configure/defaults/main.yml

```diff
@@ -95,12 +98,18 @@ zowe_xmem_stc_group: ZWEADMIN
 
 # optional, default to ansible_ssh_host
 # required for zD&T (external IP is different from internal IP)
+zowe_zos_host:
+# optional, default to zowe_zos_host
+# for z/OS deployment, this can be left empty and be same as zowe_zos_host
+# for all-in-one docker deployment, this can be set to domain where you start docker container
 zowe_external_domain_name:
 # optional, no default value
 # required for zD&T (external IP is different from internal IP)
 zowe_external_ip_address:
 # Restrict permissions to keystore/truststore after the certificate setup script
 zowe_lock_keystore: true
+#
+zowe_launch_components: api-mediation,files-api,jobs-api,explorer-jes,explorer-mvs,explorer-uss,zss,app-server
```

Comment:
We had split api-mediation into 4 packages. Please update this default value to empty string. I believe this way, we can make it compatible with staging, and also won't conflict the value we may change in the future. For example, adding caching-service to the list.

Reply:
cleared value